open-craft/opencraft
# Add ansible var for custom certificate templates

As part of [OC-3011](https://tasks.opencraft.com/browse/OC-3011) and [OC-3032](https://tasks.opencraft.com/browse/OC-3032) a variable needs to be set to enable custom templates for certificates. This PR just sets the variable in the ansible vars so that its set in all OCIM instances by default.

**JIRA tickets**: Needed by [OC-3011](https://tasks.opencraft.com/browse/OC-3011) and [OC-3032](https://tasks.opencraft.com/browse/OC-3032)

**Merge deadline**: None

**Testing instructions**:

1. Spin up an OCIM instance.
2. Verify that the `CUSTOM_CERTIFICATE_TEMPLATES_ENABLED` exists and is set to `true`

**Reviewers**
